tests/integration/test_source_git.py

```diff
@@ -55,14 +57,18 @@ def test_basic_local_update_without_patching(
     assert spec.get_version() == "0.1.0"
 
 
+@pytest.mark.parametrize("ref", ["0.1.0", "0.1*", "0.*"])
```

Comment:
so shouldn't we test for `None` as well here?

Reply:
In this one, since no patches are created, yes. Thanks.